scripts/installation/fedora38.sh

```diff
@@ -89,7 +89,7 @@ echo "gpgcheck=1" >> /etc/yum.repos.d/mongodb-org-6.0.repo
 echo "enabled=1" >> /etc/yum.repos.d/mongodb-org-6.0.repo 
 echo "gpgkey=https://www.mongodb.org/static/pgp/server-6.0.asc" >> /etc/yum.repos.d/mongodb-org-6.0.repo
 
-RUN dnf install -y mongodb-org
+$sudo_cmd dnf install -y mongodb-org
```

Comment:
This line has been added in from the BitBucket side (and now is also present here on GitHub as seen in the latest `main` branch)